script/entrypoint

```diff
@@ -32,7 +32,11 @@ elif [ "$1" = "query" ]; then
     # For queries that have a corresponding metadata file with public_bigquery set to
     # True, the destination_table option will be mapped to the destination table in
     # the public dataset.
-    exec script/run_query --query_file="${@: -1}" ${@:1:$#-1}
+    python script/run_query --query_file="${@: -1}" ${@:1:$#-1}
+
+    # For queries with public_json set in the metadata, the results will be
+    # exported as JSON to Cloud storage.
+    exec script/publish_public_data_json --query_file="${@: -1}" ${@:1:$#-1}
```

Comment:
This could be problematic, I think. It would be nice if first the script that runs the query and then the script that checks and publishes results as JSON could be executed. 
It would be possible to move all of the logic to `run_query`, but since that is quite a bit of code it might get harder to maintain. Having a wrapper script that calls both scripts would probably also work. I'm not sure what the best way here is.

Reply:
When we get to the point of having DAGs generated within this repo, it would indeed to nice to separate out the steps within the DAG. For the time being, it looks like the approach here is attempting to satisfy the spirit of "all you need to do to publish a query is to set public to true in the metadata". If we continue to see that as a requirement, then we have to handle it all within this single invocation of `entrypoint`.

I'm in favor of the approach captured here, though I dislike the asymmetry of invoking the second script via `exec` and not the first one. `exec` is nice for the simple case where the entrypoint is dispatching to a single script, but I think we should drop it on both invocations here.

Reply:
Unfortunately we're not at the point right now where it's ever just "specify in the metadata", since this still needs to get added to Airflow anyways. I also think there's a halfway point between "generate dags in this repo" and "run everything as a script block". At it's simplest we could specify  this as a new util that first runs queries and then publishes. Later we could merge that back into generated DAGs.

Reply:
Yeah, it might be more convenient to call this script from Airflow. I'll look into adding it as a util which gets executed after the query.

Reply:
I added something in Airflow which might work for this: https://github.com/mozilla/telemetry-airflow/pull/911